lib/rubocop/cop/rails/relative_date_constant.rb

```diff
@@ -25,6 +25,8 @@ class RelativeDateConstant < Cop
         RELATIVE_DATE_METHODS = %i[ago from_now since until].freeze
 
         def on_casgn(node)
+          return if node.children.last.lambda_or_proc?
```

Comment:
Please, use node destructuring here (e.g. something like `_scope, const_name, value = *node`) instead of accessing `node.children` directly. I don't like this, because it gives 0 indication what exactly are you checking when you flatten the nodes to an array.